ethereum/api/src/test/resources/JsonRpcHttpService/rpc_known_clients.txt

```diff
@@ -0,0 +1,3 @@
+#client certificates fingerprint obtained from rpc_client_keystore.pfx
```

Comment:
The Ethsigner work dyanmically creates this file from the PFX - might be nicer than having the file stored as a resource?

Reply:
if we are storing keystore files as resource then there is no harm in storing known_clients as a resource ... otherwise, we should generate both keystores and known_files dynamically in test case. I'll try the later to see how it looks like.